app/assets/javascripts/discourse/app/components/table-header-toggle-bulk.js

```diff
@@ -0,0 +1,26 @@
+import TableHeaderToggleComponent from "discourse/components/table-header-toggle";
+
+export default TableHeaderToggleComponent.extend({
+  click(e) {
+    const self = this;
+    const onClick = function (sel, callback) {
+      const target = $(e.target).closest(sel);
+
+      if (target.length === 1) {
+        callback.apply(self, [target]);
+      }
+    };
+
+    onClick("button.bulk-select-all", function () {
```

Comment:
Might as well make all these `() =>` functions too, since you aren't using `this`. (very minor)